Makefile

```diff
@@ -88,3 +88,6 @@ coveralls:
 circle-ci:
 	@${cidocker} make install-deps build-local check-local coveralls
 	make integration-tests
+
+start-services:
```

Comment:
You need to update the `.PHONY` target line above with this target Otherwise it is going to create an empty file calleed `start-services` in the root directory.